README.md

```diff
@@ -1,6 +1,5 @@
 db-migrate-plugin-babel
 =======================
 
-Add babel support to db-migrate via `babel-register`.
-
-Simply install 'babel-register' and this plugin and `db-migrate` should pick it up automatically.
+Simply install either `babel-register` or `@babel/register` along with `db-migarte` and this plugin
+should pick it up automatically.
```

Comment:
“Migarte” typo

Reply:
Also db-migrate picks up this plugin automatically, rather than the other way around; but this could do with better wording 👍

Reply:
Gah, spelling is the worst. I'll make those changes.